packages/main/src/MessageStrip.hbs

```diff
@@ -27,7 +27,7 @@
 			icon="decline"
 			design="Transparent"
 			class="ui5-message-strip-close-button"
-			._buttonAccInfo="{{accInfo.button}}"
+			title="{{accInfo.button.title}}"

```

Comment:
If it's just the title, you can further simplify to directly bind "_closeButtonText" and remove the get accInfo() {

Reply:
Done